### ISSUE TITLE
Argument placeholder not replaced

### ISSUE THREAD

**Issue by ihcnet** [micronaut-projects/micronaut-gradle-plugin#129]
The `__ARG__` placeholder is not replacing when using version 1.3.3 of the plugin. 

For instance:

```groovy
plugins {
    id("com.github.johnrengelman.shadow") version "6.1.0"
    id("io.micronaut.application") version "1.3.3"
}

version = "0.1"
group = "hello"

repositories {
    mavenCentral()
    jcenter()
}

micronaut {
    runtime("netty")
    testRuntime("junit5")
    processing {
        incremental(true)
        annotations("hello.world.*")
    }
}

dependencies {
    implementation("io.micronaut:micronaut-validation")
    implementation("io.micronaut:micronaut-runtime")
    implementation("io.micronaut:micronaut-http-client")
    runtimeOnly("ch.qos.logback:logback-classic")
}


application {
    mainClass.set("hello.world.Application")
}

java {
    sourceCompatibility = JavaVersion.toVersion("1.8")
    targetCompatibility = JavaVersion.toVersion("1.8")
}
```

running the command:

```bash
./gradlew dockerBuild
```

yields a Dockerfile like:

```Dockerfile
FROM openjdk:15-alpine
WORKDIR /home/app
COPY build/layers/libs /home/app/libs
COPY build/layers/resources /home/app/resources
COPY build/layers/application.jar /home/app/application.jar
EXPOSE 8080
ENTRYPOINT ["java", "__ARGS__", "-jar", "/home/app/application.jar"]
```

#### Additional Notes
* The `__ARGS__` argument is not replaced on the ENTRYPOINT instruction. 
* If `args` are specified through the task, the placeholder is still remains.
* The placeholder does not remain when using version 1.3.2.

**Comment by osh-onstructive** [comment]
There is a PR open to fix this issue.

**Comment by ilopmar** [comment]
Fixed in #127.

@ihcnet you can manually downgrade the plugin version until we release a new one with the fix.

**Comment by ilopmar** [comment]
@ihcnet I've published 1.3.4 version with the fix.
